app/containers/AudioPlayer/Seek.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { LayoutChangeEvent, View, TextInput, TextInputProps } from 'react-native';
+import { PanGestureHandler, PanGestureHandlerGestureEvent } from 'react-native-gesture-handler';
+import Animated, {
+	SharedValue,
+	runOnJS,
+	useAnimatedGestureHandler,
+	useAnimatedProps,
+	useAnimatedStyle,
+	useDerivedValue,
+	useSharedValue
+} from 'react-native-reanimated';
+
+import styles from './styles';
+import { useTheme } from '../../theme';
+import { AUDIO_BUTTON_HIT_SLOP, THUMB_SEEK_SIZE } from './utils';
+
+const DEFAULT_TIME_LABEL = '00:00';
+
+const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);
+
+interface ISeek {
+	duration: SharedValue<number>;
+	currentTime: SharedValue<number>;
+	loaded: boolean;
+	onChangeTime: (time: number) => Promise<void>;
+}
+
+// Could use from react-native-redash. We have it as dev dep already.
+function clamp(value: number, min: number, max: number) {
+	'worklet';
+
+	return Math.min(Math.max(value, min), max);
+}
+
+// https://docs.swmansion.com/react-native-reanimated/docs/2.x/fundamentals/worklets/
+const formatTime = (ms: number) => {
+	'worklet';
+
+	const minutes = Math.floor(ms / 60);
+	const remainingSeconds = Math.floor(ms % 60);
+	const formattedMinutes = String(minutes).padStart(2, '0');
+	const formattedSeconds = String(remainingSeconds).padStart(2, '0');
+	return `${formattedMinutes}:${formattedSeconds}`;
+};
+
+const Seek = ({ currentTime, duration, loaded = false, onChangeTime }: ISeek) => {
+	const { colors } = useTheme();
+	const maxWidth = useSharedValue(1);
+	const translateX = useSharedValue(0);
+	const timeLabel = useSharedValue(DEFAULT_TIME_LABEL);
+	const scale = useSharedValue(1);
+
+	const styleLine = useAnimatedStyle(() => ({
+		width: translateX.value
+	}));
+
+	const styleThumb = useAnimatedStyle(() => ({
+		transform: [{ translateX: translateX.value - THUMB_SEEK_SIZE / 2 }, { scale: scale.value }]
+	}));
+
+	const onLayout = (event: LayoutChangeEvent) => {
+		const { width } = event.nativeEvent.layout;
+		maxWidth.value = width;
+	};
+
+	const onGestureEvent = useAnimatedGestureHandler<PanGestureHandlerGestureEvent, { offsetX: number }>({
+		onStart: (_, ctx) => {
+			ctx.offsetX = translateX.value;
+		},
+		onActive: ({ translationX }, ctx) => {
+			translateX.value = clamp(ctx.offsetX + translationX + (THUMB_SEEK_SIZE / 2) * Math.sign(translationX), 0, maxWidth.value);
+			scale.value = 1.3;
+			currentTime.value = (translateX.value * duration.value) / maxWidth.value || 0;
+		},
+		onEnd: () => {
+			scale.value = 1;
+			runOnJS(onChangeTime)(Math.round(currentTime.value * 1000));
+		}
+	});
+
+	useDerivedValue(() => {
+		const timeInProgress = (currentTime.value * maxWidth.value) / duration.value || 0;
+		translateX.value = timeInProgress;
+		timeLabel.value = formatTime(currentTime.value);
+	}, [translateX, maxWidth, duration, currentTime]);
```

Comment:
https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/105f83a6-0d14-4964-8953-478eaa5991e0

This is the problem that isPanning was fixing